webapp/javascript/components/FlameGraph/FlameGraphRenderer.jsx

```diff
@@ -259,6 +260,14 @@ class FlameGraphRenderer extends React.Component {
         })}
       >
         <div className="canvas-container">
+          {this.props.uploader ? (
```

Comment:
Do we really need this here? Can we move this up to the parent component?

I don't like it having here, since for all effects the `FlamegraphRenderer` is the main Flamegraph component, as in this is the component other users* will interact with.
*By others users I mean the grafana plugin, eventually in the documentation, etc.

Reply:
Right, with the rest of components out, there's no reason to keep the file uploader there either! Changed at https://github.com/pyroscope-io/pyroscope/pull/580/commits/165e7112c970d7c19c844f720814794658f0dd9e